lib/HTFeed/Volume.pm

```diff
@@ -244,7 +245,11 @@ sub get_checksum_md5 {
 
     my $checksums = {};
 
-    open(FILE, $checksum_path) or die $!;		
+    unless ( -e $checksum_path ) {
+      $self->set_error("MissingFile", file => $checksum_file);
+    }
+
+    open(FILE, $checksum_path) or die($!);
```

Comment:
We could also have changed this to `die("Can't open $checksum_path: $!")`, which still might be better